
heartbeat_detector:
  ros__parameters:
    grpc:
      address: "unix:/run/plcnext/grpc.sock"
      type: "tcp"
    heartbeat_bool: "Arp.Plc.Eclr/MainInstance.xHeartbeat"
    heartbeat_timeout_s: 5 # Timeout in seconds. If no heartbeat recevied in this interval, shout a warning.
    poll_rate_hz: 10 # Poll rate in Hz to check the bool
